third-party/chpl-venv/Makefile

```diff
@@ -5,15 +5,13 @@ endif
 CHPL_MAKE_HOST_TARGET = --host
 include $(CHPL_MAKE_HOME)/make/Makefile.base
 
-PYTHON = $(shell which python)
+# do not include ./Makefile.include from Makefile.base
```

Comment:
Or I'd remove this comment, and instead stuff a comment in chpl-venv/Makefile.include about why it isn't/shouldn't be included in Makefile.base